ci/ray_ci/docker_container.py

```diff
@@ -51,8 +52,10 @@ def _get_image_version_tags(self) -> List[str]:
         branch = os.environ.get("BUILDKITE_BRANCH")
         sha_tag = os.environ["BUILDKITE_COMMIT"][:6]
         pr = os.environ.get("BUILDKITE_PULL_REQUEST", "false")
+
         if branch == "master":
-            return [sha_tag, "nightly"]
+            formatted_date = datetime.now().strftime("%y%m%d")
+            return ["nightly", f"nightly.{formatted_date}"]
```

Comment:
actually, in addition to adding the date into the tag, could you also add the commit digest (sha)? like `nightly.{formatted_date}.{sha_tag}` ? this will enable us to run nightly for more than once a day but without overwriting artifacts from different commits.

also, we will have both commit and date in the tag, which makes it:

- easy to sort by time
- also easy to tell the source it is generated from

Reply:
That makes sense! I changed to this format and also use `nightly.{formatted_date}.{sha}` as first element so it's grabbed as canonical tag as well